src/peerbook.ts

```diff
@@ -69,17 +68,11 @@ export class PeerbookConnection {
 
     async adminCommand(cmd: unknown): Promise<string> {
         return new Promise((resolve, reject) => {
```

Comment:
Consider adding `async` to the function passed to the `Promise` and using `await` instead of calling `then` and using `try` - `catch` instead of calling `catch`